etcdctl/ctlv2/command/util.go

```diff
@@ -78,7 +79,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "http://127.0.0.1:2379,http://127.0.0.1:4001"
```

Comment:
It should really be kept in common if possible. I'm not really convinced this is a sane fix though. 127.0.0.1 should work everywhere. The original issue was about the etcd server not binding correctly, so this seems like it's only masking the problem?